src/collective/mustread/tracker.py

```diff
@@ -91,11 +103,117 @@ def most_read(self, days=None, limit=None):
         for record in self.query_all(query):
             yield api.content.get(UID=record.uid)
 
-    def schedule_must_read(self, obj, userids=None, deadline=None):
-        raise NotImplementedError()
+    def schedule_must_read(self, obj, userids, deadline):
+        # get existing must read items for this object
+        qry = self._read(uid=utils.getUID(obj))
+        existing_users = [m.userid for m in qry.all()]
+
+        path = '/'.join(obj.getPhysicalPath())
+        uid = utils.getUID(obj)
+        hostname = utils.hostname()
+        now = datetime.utcnow()
+        current_user = api.user.get_current().id
```

Comment:
you are right, i changed scheduled_by to be explicitly passed as argument to schedule_must_read